signer/src/ecdsa.rs

```diff
@@ -587,4 +609,119 @@ mod tests {
 
         assert_eq!(signed_message_v2, original_proto);
     }
+
+    #[test]
+    fn poc_bypass_sign2() {
```

Comment:
nit: better name?

Reply:
Agreed, I'll think of something.

Reply:
Done in https://github.com/stacks-network/sbtc/pull/1188/commits/b587d01aae07b48128c710896bcb519f82cd3c1e.